docs/CODING_STYLE.rst

```diff
@@ -67,32 +67,219 @@ Coding guide
 
 Comments
 --------
+Before every function prototype in ``.h`` files, include a Doxygen-formatted comment
+like this:
 
-Before every function have in ``.h`` files a comment like this:
+.. code-block:: c
 
-.. code:: c
+    /**
+     * Brief description
+     *
```

Comment:
No blank lines 🙂

Reply:
> No blank lines 🙂

Hi, Gabor.  Can you clarify what is meant here?  Do you mean you want something like this:

```
/**
 * Set how to place (where to align) the items and tracks
 *
 * @param  obj                 pointer to an object. The parent must have flex layout else nothing will happen.
 * @param  main_place          where to place the items on main axis (in their track). Any value of `lv_flex_align_t`.
 * @param  cross_place where   to place the item in their track on the cross axis. `LV_FLEX_ALIGN_START/END/CENTER`
 * @param  track_cross_place   where to place the tracks in the cross direction. Any value of `lv_flex_align_t`.
 *
 * @note  The resulting behavior is very similar to flex containers in CSS.
 */
void lv_obj_set_flex_align(lv_obj_t * obj, lv_flex_align_t main_place, lv_flex_align_t cross_place,
                           lv_flex_align_t track_cross_place);
```
to be this instead?
```
/**
 * Set how to place (where to align) the items and tracks
 * @param  obj                 pointer to an object. The parent must have flex layout else nothing will happen.
 * @param  main_place          where to place the items on main axis (in their track). Any value of `lv_flex_align_t`.
 * @param  cross_place where   to place the item in their track on the cross axis. `LV_FLEX_ALIGN_START/END/CENTER`
 * @param  track_cross_place   where to place the tracks in the cross direction. Any value of `lv_flex_align_t`.
 * @note  The resulting behavior is very similar to flex containers in CSS.
 */
void lv_obj_set_flex_align(lv_obj_t * obj, lv_flex_align_t main_place, lv_flex_align_t cross_place,
                           lv_flex_align_t track_cross_place);
```

Reply:
Yes, I meant that. Just because we don't have empty lines after the brief now, so it would be strange to ask the opposite.

Reply:
> Yes, I meant that. Just because we don't have empty lines after the brief now, so it would be strange to ask for the opposite.

That completely makes sense to me.  **thumbs up**